src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int WORKED_HOURS_INDEX = 2;
+    private static final int SALARY_PER_HOUR_INDEX = 3;
+    private static final DateTimeFormatter TIME_PATTERN = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder()
+                .append("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
```

Comment:
Every strange symbol better makes a constant variable and gives an informative name (What is it?). Especially if you use it several times.